### ISSUE TITLE
Pictograms: Certain pictograms are missing

### ISSUE THREAD

**Issue by joshblack** [carbon-design-system/carbon#5824]
@fabianklonsdorf commented on [Tue Apr 07 2020](https://github.com/carbon-design-system/carbon-website/issues/1099)

## Detailed description

Java, JS, Mas, Mqa, SSO pictos are missing and instead displayed as api picto. 
The icons for the above mentioned topics should be displayed correctly.
Firefox, Chrome

## Steps to reproduce the issue

1. Step one: Go to carbondesignsystem.com
2. Step two: Click on Guidelines and then to pictograms
3. Step three: Go to one of the icons shown above
4. See the issue

## Additional information

<img width="1289" alt="Screenshot 2020-04-07 at 13 41 03" src="https://user-images.githubusercontent.com/47037236/78681230-be631180-78ec-11ea-856a-075c5fd52979.png">

---

@conradennis commented on [Tue Apr 07 2020](https://github.com/carbon-design-system/carbon-website/issues/1099#issuecomment-610473555)

Same issue for IDL site: https://github.com/carbon-design-system/design-language-website/issues/501

looks like these should all have plex abbreviations in the boxes, screenshots from the picto repo:
<img width="171" alt="Screen Shot 2020-04-07 at 12 00 10 PM" src="https://user-images.githubusercontent.com/16782944/78692122-6e258880-78c7-11ea-81cb-36f03f8bf400.png">

<img width="194" alt="Screen Shot 2020-04-07 at 12 00 25 PM" src="https://user-images.githubusercontent.com/16782944/78692125-6ebe1f00-78c7-11ea-8930-0178dc21a9f8.png">

---

@conradennis commented on [Tue Apr 07 2020](https://github.com/carbon-design-system/carbon-website/issues/1099#issuecomment-610478431)

@joshblack can you take a look when you get a chance?

**Comment by vpicone** [comment]
Looks like `<text>` element contents are being stripped from these pictograms (and potentially some icons?) before they get to IDL/Carbon libraries. 

Once we solve for that, the `fontFamily` attribute being strictly `IBMPlexSans` creates a weird dependency on font loading. Users will be required to have the font installed locally or precisely in their web stylesheet in order for the text to load. We might want to insist that the pictograms be re-exported rasterized rather than using the `text` element

**Comment by joshblack** [comment]
cc @dudley-ibm @laurenmrice is it possible that assets can be exported with the text expanded into paths instead of with the font family? Right now it looks like the assets include the `font` glyphs in the SVG itself which is causing some size concerns and are stripped away by the optimization process as a result

Example

![image](https://user-images.githubusercontent.com/3901764/87712699-b1b54b80-c76e-11ea-87fb-9342bc696aed.png)

**Comment by dudley-ibm** [comment]
@joshblack - yes.
They will all be fixed when I finish the mass-upload we talked about in the last meeting for the breaking change (outlined artwork/32x32 px pasteboard/downloadable asset changed from rendered version to BXD provided version).

If I send you a version now of those in question with only that text portion expanded into paths, they will still display incorrectly based on what your rendering process is doing to compound graphics (adding an unnecessary stroke to outlined shapes).

**Comment by vpicone** [comment]
Potentially related: https://github.com/carbon-design-system/carbon/issues/6498

Looks like when assets _are_ exported using paths we might be mangling it a bit with SVGO. We might prob need to make sure once assets are exported using the outline setting that we're not doing anything weird to mess with it.

**Comment by dudley-ibm** [comment]
I'm anxiously awaiting a breaking change that should fix these and any other rendering issues. I've resubmitted updated artwork for each and every pictogram in an effort to rectify this problem that exists across the entire library. Apparently the entire approach to how the SVG filess render is changing, and users will soon be able to download the SVG exactly as I've provided it, not a version rendered out by Sketch.
